Update recover from ransomware (#6392)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/recover-from-ransomware.md

- [Microsoft Defender for Office 365 plan 1 and plan 2](defender-for-office-365.md)
- [Microsoft 365 Defender](../defender/microsoft-365-defender.md)

> Even if you take every precaution to protect your organization, you can still fall victim to a [ransomware](/windows/security/threat-protection/intelligence/ransomware-malware) attack. Ransomware is big business, and in today's threat landscape Microsoft 365 is an ever increasing [target for sophisticated attacks](https://i.blackhat.com/USA21/Wednesday-Handouts/us-21-Cloudy-With-A-Chance-Of-APT-Novel-Microsoft-365-Attacks-In-The-Wild.pdf).

Reviewer: ```suggestion
Even if you take every precaution to protect your organization, you can still fall victim to a [ransomware](/windows/security/threat-protection/intelligence/ransomware-malware) attack. Ransomware is big business, and in today's threat landscape Microsoft 365 is an ever-increasing [target for sophisticated attacks](https://i.blackhat.com/USA21/Wednesday-Handouts/us-21-Cloudy-With-A-Chance-Of-APT-Novel-Microsoft-365-Attacks-In-The-Wild.pdf).
```

- It's important for you respond quickly to the attack and its consequences. The longer you wait, the less likely it is that you can recover the affected data.

## Step 1: Verify your third-party backups

> The Microsoft 365 compliance suite of tools are designed to meet your organizations [legal and regulatory standards](/microsoft-365/compliance/) which is why as a best practice, we [recommend](https://www.microsoft.com/servicesagreement) that you regularly backup your Microsoft 365 Content and Data using Third-Party Apps and Services.

Reviewer: ```suggestion
The Microsoft 365 compliance suite of tools is designed to meet your organization's [legal and regulatory standards](/microsoft-365/compliance/) which is why as a best practice, we [recommend](https://www.microsoft.com/servicesagreement) that you regularly back up your Microsoft 365 content and data using third-party apps and services.
```

> In order to ensure you have the ability to restore your data, it's important to regularly test your backups and validate they can be restored in the event of an attack. In the case of an actual attack, these restorations should not be completed until **after** you've removed the ransomware payload (malware) from your environment and **after** you have verified there is no unauthorized access in your Microsoft 365 environments.

Reviewer: ```suggestion
In order to ensure you have the ability to restore your data, it is important to regularly test your backups and validate they can be restored in the event of an attack. In the case of an actual attack, these restorations should not be completed until **after** you've removed the ransomware payload (malware) from your environment and **after** you have verified there is no unauthorized access in your Microsoft 365 environments.
```

> To ensure your backups were not also affected by ransomware, the third-party service should be air gapped from your environments.

Reviewer: ```suggestion
To ensure your backups were not also affected by ransomware, the third-party service should be air-gapped from your environments.
```
